AUTHORS

```diff
@@ -1,4 +1,6 @@
 Super Grub Disk authors:
 
 Adrian Gibanel Lopez designed and implemented the initial version.
-Jordan Uggla improved Super Grub2 Disk source code.
+Jordan Uggla improved the Super Grub2 Disk source code.
```

Comment:
This should be on a separate commit. So that reverting the functionality change does not revert you being an author.

Reply:
✔